src/components/block/index.ts

```diff
@@ -9,24 +9,25 @@ import {
   PopoverItemParams
 } from '../../../types';
 
-import { SavedData } from '../../../types/data-formats';
+import type { SavedData } from '../../../types/data-formats';
 import $, { toggleEmptyMark } from '../dom';
 import * as _ from '../utils';
-import ApiModules from '../modules/api';
+import type ApiModules from '../modules/api';
 import BlockAPI from './api';
 import SelectionUtils from '../selection';
-import BlockTool from '../tools/block';
+import type BlockTool from '../tools/block';
 
-import BlockTune from '../tools/tune';
-import { BlockTuneData } from '../../../types/block-tunes/block-tune-data';
-import ToolsCollection from '../tools/collection';
+import type BlockTune from '../tools/tune';
+import type { BlockTuneData } from '../../../types/block-tunes/block-tune-data';
+import type ToolsCollection from '../tools/collection';
 import EventsDispatcher from '../utils/events';
-import { TunesMenuConfigItem } from '../../../types/tools';
+import type { TunesMenuConfigItem } from '../../../types/tools';
```

Comment:
why not use new `@/types/` alias everywhere?

Reply:
It is out of scope. 
I changed only places I touched. There are too many files in the pr already.